third_party/mbedtls/repo/library/entropy_poll.c

```diff
@@ -50,6 +50,28 @@
 
 #if defined(_WIN32) && !defined(EFIX64) && !defined(EFI32)
 
+#if defined(_KERNEL_MODE)
+
+#include <ntdef.h>
+#include <bcrypt.h>
+
+int mbedtls_platform_entropy_poll( void *data, unsigned char *output, size_t len,
```

Comment:
Doing a [search for mbedtls_platform_entropy_poll](https://github.com/openthread/openthread/search?utf8=%E2%9C%93&q=mbedtls_platform_entropy_poll) shows that it is called directly as far as I can tell. Meaning, if I don't make this change, we will link errors for the user mode code the other implementation is calling. Is there anyway around that?